Production/python/Production.py

```diff
@@ -51,7 +55,11 @@
 process.MessageLogger.cerr.FwkReport.reportEvery = 100
 
 process.load('Configuration.StandardSequences.MagneticField_cff')
-process.load('Configuration.Geometry.GeometryRecoDB_cff')
+# include Phase2 specific configuration only after 11_1_X
+if int(cmssw_release_numbers[0]) >= 11 and int(cmssw_release_numbers[1]) >= 1:
```

Comment:
Better to define a boolean `isPhase2 = int(cmssw_release_numbers[0]) >= 11 and int(cmssw_release_numbers[1]) >= 1` and use it here and in L97. It will make the condition more transparent and easier to adjust in the future, if needed.

Reply:
Ok, is now updated